### ISSUE TITLE
Station blue prints not showing Structural data

### ISSUE THREAD

**Issue by carshalash** [tgstation/tgstation#58810]
## Testmerges:

#58631 

## Reproduction:

Attempt to use cyborg station blueprints to show station damage
Toggle on and off repeatedly when nothing shows up
Ask admins to test with the CE blueprints
Same thing happens

**Comment by MrMelbert** [comment]
`[01:06:32] Runtime in blueprints.dm, line 120: type mismatch: 166 - "19x15"`
```proc name: get images (/obj/item/areaeditor/blueprints/proc/get_images)
usr: Mr Melbert/(Sits-At-Centcom)
usr.loc: (Thunderdome Arena (166,49,1))
src: the station blueprints (/obj/item/areaeditor/blueprints)
src.loc: Sits-At-Centcom (/mob/living/carbon/human)
call stack:
the station blueprints (/obj/item/areaeditor/blueprints): get images(the floor (166,49,1) (/turf/open/floor/iron), "19x15")
the station blueprints (/obj/item/areaeditor/blueprints): set viewer(Sits-At-Centcom (/mob/living/carbon/human), "<span class=\'notice\'>You fli...")
the station blueprints (/obj/item/areaeditor/blueprints): Topic("src=\[0x2040e25];view_blueprin...", /list (/list))```

**Comment by Sealed101** [comment]
fixed by #82565
